decoration.rb

```diff
@@ -0,0 +1,12 @@
+equire_relative './nameable'
```

Comment:
* Kindly fix this typo here instead of `require_relative` you have `equire_relative`